website/quickfiles/views.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+import os
+from flask import send_from_directory
+from website.settings import EXTERNAL_EMBER_APPS
+
+quickfiles_dir = os.path.abspath(os.path.join(os.getcwd(), EXTERNAL_EMBER_APPS['quickfiles']['path']))
+
+def use_ember_app(**kwargs):
```

Comment:
Also need to add `if PROXY_EMBER_APPS:` here

Reply:
ah